### PR TITLE
FXVPN-233: Audit and update strings

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Removes hard-coded strings 
- Updates existing strings to latest

### CODE REVIEW

**Review by strseb — APPROVED** [review]

File: src/_locales/en/messages.json [hunk 1]
```diff
@@ -190,6 +186,98 @@
...
+  "onboarding1_body": {
+    "message": "With the Firefox extension, you can turn off VPN protection for specific websites, while keeping it on for everything else.",
+    "description": "Body text in onboarding panel"
+  },
+  "onboarding2_title": {
+    "message": "See different websites from different locations",
+    "description": "Title in onboarding panel"
+  },
+  "onboarding2_body": {
+    "message": "",
```
Comment (flodolo) [inline src/_locales/en/messages.json:222]: @lesleyjanenorton 
This string is empty.

I think it should be `News from London. Sports from Sao Paulo. Shopping from Tokyo. Set website-specific locations for personalized browsing.`

File: src/_locales/en/messages.json [hunk 2]
```diff
@@ -190,6 +186,98 @@
...
+    "description": "Body text in onboarding panel"
+  },
+  "onboarding3_title": {
+    "message": "Keep protection for Firefox “on”",
+    "description": "Title in onboarding panel"
+  },
+  "onboarding3_body": {
+    "message": "Use the extension to turn on VPN protection for Firefox — even when protection is off for the rest of your computer. ",
```
Comment (flodolo) [inline src/_locales/en/messages.json:230]: Extra space at the end of the string.

File: src/_locales/en/messages.json [hunk 3]
```diff
@@ -190,6 +186,98 @@
...
+  "noSignal": {
+    "message": "No Signal",
+    "description": "Indicator text describing the status of the VPN"
+  },
+  "unstable": {
+    "message": "Unstable",
+    "description": "Indicator text describing the status of the VPN"
+  },
+  "connecting": {
+    "message": "Connecting...",
```
Comment (flodolo) [inline src/_locales/en/messages.json:266]: Use `…` (single unicode character).

**Comment by flodolo** [comment]
Opened https://github.com/mozilla-extensions/mozilla-vpn-extension/pull/106

**Pull request merged**